codecov_auth/models.py

```diff
@@ -510,10 +510,11 @@ class GithubAppInstallation(
 
     def is_configured(self) -> bool:
         """Returns whether this installation is properly configured and can be used"""
-        if self.name == GITHUB_APP_INSTALLATION_DEFAULT_NAME:
-            # The default app is configured in the installation YAML
+        if self.app_id is not None and self.pem_path is not None:
```

Comment:
Is there a way to tell if it's configured properly? Idk if they could end up with a self.app_id that isn't related to their PEM

Reply:
There is no way to tell that, I don't think.
If it's configured improperly we'd see the app being unable to be used, and errors everywhere.

But the configuration needs to be done manually by us. Low chance of someone that doesn't know what their doing making this config.